sys/include/net/gcoap.h

```diff
@@ -373,15 +400,23 @@ extern "C" {
  * @brief   Server port; use RFC 7252 default if not defined
  */
 #ifndef GCOAP_PORT
+#ifdef  MODULE_SOCK_DTLS
+#define GCOAP_PORT              (5684)
+#else
 #define GCOAP_PORT              (5683)
-#endif
+#endif  /* MODULE_SOCK_DTLS */
+#endif  /* GCOAP_PORT */
 
 /**
  * @brief   Size of the buffer used to build a CoAP request or response
```

Comment:
I expected to see a comment similar to what was in `Makefile.dep`. The additional 128 bytes deserves some explanation so the user can change it if that is reasonable.

> Payload size might be bigger when using DTLS with ECC, causing errors not present
> when using plain CoAP

Reply:
Rephrased and added the comment in dfdbb86c0